consensus/cryptarchia-engine/src/lib.rs

```diff
@@ -325,21 +325,21 @@ pub mod tests {
     #[test]
     fn test_getters() {
         let engine = Cryptarchia::from_genesis([0; 32], config());
-        let parent = engine.genesis();
+        let id_0 = engine.genesis();
 
         // Get branch directly from HashMap
         let branch1 = engine
             .branches
-            .get(&parent)
-            .ok_or("At least one branch should be there")
+            .get(&id_0)
+            .ok_or("at least one branch should be there")
             .expect("branch1 is not set");
 
         let branches = engine.branches();
 
         // Get branch using getter
         let branch2 = branches
-            .get(&parent)
-            .ok_or("At least one branch should be there")
+            .get(&id_0)
+            .ok_or("at least one branch should be there")
```

Comment:
nit: you can call `expect` on an `Option` as well

Reply:
Simplified at https://github.com/logos-co/nomos-node/pull/657/commits/20915d23371476c42d962e7cd601ac499c26960c